openapi.yaml

```diff
@@ -14656,6 +15387,168 @@ components:
+        # sub-properties.
+        label:
+          x-linode-filterable: true
+          x-linode-cli-display: 2
+          example: lkecluster12345
+        region:
+          readOnly: true
+          x-linode-filterable: true
+          x-linode-cli-display: 3
+          example: us-central
+        version:
+          x-linode-filterable: true
+          example: "1.16"
+        tags:
+          x-linode-filterable: true
+          example:
+          - ecomm
+          - blogs
+    LKENodePoolRequestBody:
+      type: object
+      description: >
+        Specifies a collection of Linodes which will be members of a Kubernetes
+        cluster.
+      properties:
+        type:
+          description: A Linode Type for all of the nodes in the Node Pool.
+          example: "g6-standard-4"
+        count:
+          type: integer
+          description: The number of nodes in the Node Pool.
+          minimum: 1
+          maximum: 100
+          example: 6
+    LKENodePool:
+      # Note that LKENodePool inherits all properties of
+      # LKENodePoolRequestBody and then overrides some of the sub-properties
+      type: object
+      allOf:
+      - $ref: '#/components/schemas/LKENodePoolRequestBody'
+      description: >
+        The set of Node Pools which are members of the Kubernetes cluster.
+        Node Pools consist of a Linode type, the number of Linodes to
+        deploy of that type, and additional status information.
+      properties:
+        id:
+          type: integer
+          description: >
+            This Node Pool's unique ID.
+          example: 456
+        linodes:
+          type: array
+          description: >
+            Status information for the Linodes which are members of this Node
+            Pool. If a Linode has not been provisioned for a given Node slot,
+            its ID will be returned as null.
+          items:
+            $ref: '#/components/schemas/LKELinodeStatus'
+    LKELinodeStatus:
+      type: object
+      description: >
+        Status information for a Linode which is a member of a Kubernetes
+        cluster.
+      readOnly: true
+      properties:
+        id:
+          type: number
+          description: >
+            The Linode's ID. When no Linode is currently provisioned for this
+            Node slot, this ID will be null.
+          x-linode-filterable: true
+          example: 123456
+        status:
+          type: string
+          description: >
+            The Linode's status as it pertains to being a Kubernetes node.
+          enum:
+          - ready
+          - not_ready
+          example: ready
+    LKEVersion:
+      type: object
+      description: >
+        LKE versions
+      readOnly: true
+      properties:
+        id:
+          type: string
+          description: >
+            A Kubernetes version number available for deployment
+            to a Kubernetes cluster.
+          example: "1.16"
```

Comment:
The curl example response for this one should use this string rather than show an integer 1234